src/varlinkctl/varlinkctl.c

```diff
@@ -94,6 +94,9 @@ static int parse_argv(int argc, char *argv[]) {
         assert(argc >= 0);
         assert(argv);
 
+        if (argv_looks_like_help(argc, argv))
```

Comment:
Hmm, varlinkctl already has `verb_help`, so just make that the default verb instead?

But I like the current behavior better, which allows us to define a proper default verb in the future.

Reply:
ok